pipe/pipe.py

```diff
@@ -22,7 +22,7 @@ class OWASPDependencyCheck(Pipe):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.scan_path = self.get_variable('SCAN_PATH')
-        self.suppression_path = self.get_variable('SUPPRESSION_FILE_PATH')
+        self.suppression_path = self.get_variable('SUPPRESSION_FILE_PATH') if self.get_variable('SUPPRESSION_FILE_PATH') else '/build/suppression.xml'
```

Comment:
Can we add an additional check to see if the file exists on line 46, so that it doesn't add --suppression if it doesn't.

Reply:
i've added that check